duplicate_private_upload_ids/README.md

```diff
@@ -0,0 +1,52 @@
+These scripts report and fix duplicate private uploads ids.  
+
+Note: Backup the Mongo database before executing these
+
+In a terminal on the mongo master node for the specified environment, execute the following commands:
+
+```
+# Set ENVIRONMENT (prod|staging|devel|test|local)
```

Comment:
It was the environment I had stood up for my testing.  Basically, it is an environment like prod, but does not have a custom SSL-enabled build of mongo.